streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/SubscriptionInfoTest.java

```diff
@@ -239,9 +242,10 @@ public void generatedVersion3To6ShouldDecodeLegacyFormat() {
                 version,
                 LATEST_SUPPORTED_VERSION,
                 processId,
-                activeTasks,
-                standbyTasks,
-                "localhost:80"
+                ACTIVE_TASKS,
+                STANDBY_TASKS,
+                "localhost:80",
+                null
```

Comment:
Q: Why do you pass in `null` here but in some other cases you pass in `TASK_LAGS`?